test/bitwise_repro/bwr_utils.hpp

```diff
@@ -503,4 +501,4 @@ class TokenHelper
 
 } // end namespace bwr_utils
 
-#endif // BRW_UTILS_HPP
```

Comment:
missing newline at end of file, otherwise looks good to me.

Reply:
Fixed, thanks.